CurrentTemplates v0.1.0

lk0001 opened the pull request:
#### Description
Simple module displaying your current build and equipment template names from GW2 API.

*Requires BlishHUD v0.11.5* (unreleased as of 12.02.2022)

#### Screenshots
(see top left corner, between icons and party menu)
![CTv0.1.0](https://user-images.githubusercontent.com/145405/153717114-8bb6e815-1fa7-41d9-8f11-c62456abc3ff.png)

#### Release
https://github.com/lk0001/Blish-HUD-Current-Templates/releases/tag/v0.1.0

dlamkins commented:
Excited to see you got your module ready for release!

Looks like the manifest you've pushed is actually a module manifest, though.  There are two types of manifests:
- Module manifests (which get bundled up inside of your bhm)
- Package manifests (which are used in the module repo to know where to download your module from)

The package manifests can be auto generated for you based on your release URL, though.  Here are the instructions on how to do it:
https://blishhud.com/docs/modules/overview/release

It mostly boils down to:
1. Installing the `BhudLib` module from the powershell gallery (`install-module bhudlib` in PowerShell to do it very quickly)
2. Import the `BhudLib` module (`import-module bhudlib`)
3. Getting the manifest details (`$pkg = Build-BhudPkgManifest -Url "https://github.com/lk0001/Blish-HUD-Current-Templates/releases/download/v0.1.0/Blish.HUD.Current.Templates_0.1.0.bhm"`)
4. Saving the manifest details to the repo (`Save-BhudPackage -Pkg $pkg -RepoRoot ".\manifests"`)

Then commit those changes and re-push.  🙂

lk0001 commented:
Argh, I didn't read instructions closely enough! Updated the manifest. I changed autogenerated name from "Blish HUD Current Templates" to just "Current Templates" though, I hope it's fine.

dlamkins commented:
Great!  I'll run the auto reviewer tonight when I return home. 👍

> I changed autogenerated name from "Blish HUD Current Templates" to just "Current Templates" though, I hope it's fine.

Not a problem!  It will display as "Current Templates" in the repo, but just know that if the manifest shows "Blish HUD Current Templates" that is how it will show once they've installed the module. 

That can be easily fixed in any future releases you do if you with to make them match. 🙂

lk0001 commented:
The manifest in the module has just `Current Templates`, so hopefully it'll show up correctly :)

dlamkins commented:
Okay, great!

On Sat, Feb 12, 2022, 7:39 PM Łukasz Kornek ***@***.***>
wrote:

> The manifest in the module has just Current Templates, so hopefully it'll
> show up correctly :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/blish-hud/bhud-pkgs/pull/110#issuecomment-1037617781>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAO4HAXJDDWTIF7Q4GLTIITU234SZANCNFSM5OHCNQ7A>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
